### DISCUSSION TITLE
Microsoft MRTK Team - thank you for all your work, sorry you were laid off :(

### DISCUSSION THREAD

**Discussion by MSDNAndi** [microsoft/MixedRealityToolkit-Unity#11390] [General]
Dear Microsoft MRTK Team,
Sorry you were all laid off by Microsoft.
It’s so sad on a personal level.

Thank you for all your work and passion over the years.

**Comment by peterneil-bimholoview** [comment]
I know this its probably too soon but do we know what's going to happen to the MRTK now?

**Reply by NoTuxNoBux** [reply]
https://github.com/microsoft/MixedRealityToolkit-Unity/issues/11391

**Reply by Zee2** [reply]
The team will be sharing an update as soon as we can. There's quite a lot of stuff on our plates right now as you might expect 😅 Stay tuned.

**Reply by MSDNAndi** [reply]
Completely understand. I have been trying to advocate for y’all. Still view it as a strategic mistake to disband the team.

**Comment by Victor-x1** [comment]
**On Twitter Zeez2 said possible Open Source MIT License.
https://twitter.com/FNS_Developer/status/1616950687010807809**

**Reply by Zee2** [reply]
The project is already MIT licensed.
